lib/glific/third_party/sheets/sheet_data.ex

```diff
@@ -0,0 +1,139 @@
+defmodule Glific.Sheets.SheetData do
+  @moduledoc """
+  The minimal wrapper for the base Sheet structure
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+  import Ecto.Query
+
+  alias __MODULE__
+
+  alias Glific.{
+    Partners.Organization,
+    Repo,
+    Sheets.ApiClient,
+    Sheets.Sheet
+  }
+
+  @required_fields [
+    :key,
+    :data,
+    :sheet_id,
+    :organization_id,
+    :synced_at
+  ]
+
+  @type t() :: %__MODULE__{
+          __meta__: Ecto.Schema.Metadata.t(),
+          id: non_neg_integer | nil,
+          key: String.t() | nil,
+          data: map() | nil,
```

Comment:
should we rename it to **row_data** or **row** ?  It matches with the sheet terminology. 
```suggestion
          data: map() | nil,
```

Reply:
@pankaj-ag make sense. updated same